### PR TITLE
Turns python-legacy CI back on in circle, fixes python-legacy test

### PR DESCRIPTION
**Pull request by spacether**
Turns python-legacy CI back on in circle, fixes python-legacy test
python-legacy testing was no longer active on PR circleCI
It is on on master branch CI and we saw a master branch failure at: https://app.travis-ci.com/github/OpenAPITools/openapi-generator/builds/244792765
@Amelia0193 @wing328 

This PR is a fix for a test failure introduced in: https://github.com/OpenAPITools/openapi-generator/pull/11140

### PR checklist
 
- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [ ] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [ ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (5.3.0), `6.0.x`
- [ ] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

File: samples/openapi3/client/petstore/python-legacy/test/test_fake_api.py [hunk 1]
```diff
@@ -13,7 +13,10 @@
 from __future__ import absolute_import
 
 import unittest
-from unittest.mock import patch
```
Comment (spacether) [inline samples/openapi3/client/petstore/python-legacy/test/test_fake_api.py:16]: This import did not work for python 2.7

**Comment by wing328** [comment]
Travis CI tests passed via https://app.travis-ci.com/github/OpenAPITools/openapi-generator/builds/244879298

**Comment by spacether** [comment]
@wing328 but my circleci tests are still broken; I haven't gotten the python 2.7 version working right...

**Comment by wing328** [comment]
Ah sorry didn't notice that. CircleCI is already failing in the master due to changes in the typescript clients.

I'll comment out https://github.com/OpenAPITools/openapi-generator/pull/11326/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1286 for the time being. Please file a new PR to re-enable it when it's ready 🙏

**Pull request merged**